packages/core/test/menu/menuTests.tsx

```diff
@@ -293,6 +290,15 @@ describe("MenuItem", () => {
         function assertClassNameCount(className: string, count: number) {
             assert.strictEqual(childContainer.queryAll(`.${className}`).length, count, `${count}x .${className}`);
         }
+
+        function mountMenuItem(props?: IMenuItemProps, childItems?: React.ReactNode) {
+            return ReactDOM.render(
```

Comment:
awkward we're not using `enzyme` here

Reply:
@giladgray Looks like we need real DOM to properly evaluate all the real-estate-dependent dynamic-layout behavior. Also looks like you were the original test author, so feel free to tell me if that's not the case!